src/Avalonia.Controls/Platform/ITopLevelNativeMenuExporter.cs

```diff
@@ -24,6 +22,6 @@ public interface INativeMenuExporterProvider
     
     public interface ITopLevelImplWithNativeMenuExporter : ITopLevelImpl
     {
-        ITopLevelNativeMenuExporter NativeMenuExporter { get; }
+        ITopLevelNativeMenuExporter? NativeMenuExporter { get; }
```

Comment:
Same

Reply:
Because of the `?` here:

https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Controls/TrayIcon.cs#L187

@danwalmsley is this correct?

Reply:
It's nullable because it might be not available for this particular toplevel. E. g. if we are running on Linux with DE that does not support the global menu. Or if global menu was disabled in platform settings (Linux, macOS)